packages/golden-layout/src/errors/ConfigurationError.js

```diff
@@ -0,0 +1,11 @@
+const ConfigurationError = function (message, node) {
+  Error.call(this);
+
+  this.name = 'Configuration Error';
+  this.message = message;
+  this.node = node;
+};
+
+ConfigurationError.prototype = new Error();
+
+export default ConfigurationError;
```

Comment:
Ugh. I guess if it works don't fix it? Would love to clean up/modernize all this code in another ticket (like there's a `utils.extend` function, and this, when we should just do `class ConfigurationError extends Error`